Dockerfile

```diff
@@ -6,7 +6,7 @@
 FROM centos:centos7
 MAINTAINER Pulp Team <pulp-list@redhat.com>
 
-RUN yum -y install http://dl.fedoraproject.org/pub/epel/beta/7/x86_64/epel-release-7-1.noarch.rpm
+RUN yum -y install http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```

Comment:
Centos7 now lets you ``yum -y install epel-release``

Can you make that change?